clients/src/main/resources/common/message/FetchRequest.json

```diff
@@ -49,7 +49,7 @@
   "fields": [
     { "name": "ReplicaId", "type": "int32", "versions": "0+",
       "about": "The broker ID of the follower, of -1 if this request is from a consumer." },
-    { "name": "MaxWait", "type": "int32", "versions": "0+",
+    { "name": "MaxWaitTimeMs", "type": "int32", "versions": "0+",
```

Comment:
Would `MaxWaitMs` be enough? The unit is typically enough to indicate it's time.

Reply:
Fair enough.